.travis.yml

```diff
@@ -29,6 +29,9 @@ virtualenv:
   system_site_packages: true
 
 install:
+  #  FIXME: remove this after merge
```

Comment:
@ThomasBinsfeld Could you remove this as merged ?

Reply:
Yes, done.